### DISCUSSION TITLE
Toucan extension tooltips aren't legible

### DISCUSSION THREAD

**Discussion by TheOct0** [darkreader/darkreader#8339] [Q&A]
Hi! First, I'd like to thank everyone who contributed to this project, it really has become something I wouldn't imagine living without.

I've run into a bit of an issue, though, when installing one of Toucan's recommended extensions (that appear during the initial installation): Toucan.

The issue is simple: the dynamic theme makes Toucan's language tooltips are rendered illegible by darkreader. This is, of course, solved by disabling it, but that makes the webpage light again, which immediately triggers a violent allergic reaction to light themes with my poor eyes.

I've tried using the dynamic theme editor with the `#toucan` CSS selector, but to no avail. Could someone help me out?

**Comment by alexanderby** [comment]
The fix for this will look like
```
*
...
CSS
.some-selector {
    background-color: var(--darkreader-neutral-background) !important;
    color: var(--darkreader-neutral-text) !important;
}
```

where `*` means the fix applies for all websites and `.some-selector` is the value of the `class="some-selector"` attribute.
